data-prepper-pipeline-parser/src/test/resources/transformation/expected/documentdb2-expected.yaml

```diff
@@ -26,82 +65,43 @@ documentdb-pipeline-reader:
         username: "${{aws_secrets:secret:password}}"
         password: "${{aws_secrets:secret:password}}"
       s3_prefix: "folder1/folder2"
-  routes:
+  route:
```

Comment:
both route and routes should work ?

Reply:
yes both will work. We could we any field name.
These only test the transformation. At this point, plugins are not loaded. The template that will be used in runtime is in this directory: data-prepper/data-prepper-pipeline-parser/src/resources/templates